src/canisters/frontend/ic-asset/src/evidence/mod.rs

```diff
@@ -37,11 +40,19 @@ pub async fn compute_evidence(
     let asset_descriptors = gather_asset_descriptors(dirs, logger)?;
 
     let canister_assets = list_assets(canister).await?;
+    let canister_asset_properties = get_assets_properties(canister, &canister_assets)
+        .await
+        .unwrap_or_default(); // older canisters don't have get_assets_properties method
```

Comment:
If there is an error retrieving asset properties, do we really want to set all properties for every asset?